src/routes/TicTacToe/+page.svelte

```diff
@@ -3,21 +3,31 @@
 	import 'papercss/dist/paper.min.css';
 
 	import { Button } from 'spaper';
-	import Game, { Status } from '../../lib/Game';
-	import { selectMove } from '../../lib/AIPlayer';
+	import Game, { GameTypes, Status } from '../../lib/Game';
+	import { selectMove as aiPlayerSelectMove } from '../../lib/AIPlayer';
```

Comment:
You can import as AIPlayer and then call `AIPlayer.selectMove`

```suggestion
	import AIPlayer  from '../../lib/AIPlayer';
```